pages/02.general/02.how-to-create-your-own-ad-filters/docs.en.md

```diff
@@ -962,12 +965,13 @@ We recommend to use the hint for disabling rule optimization. "Hint" is a specia
 ```
 Note, that you can apply multiple hints. 
 
-```
-NOT_OPTIMIZED hint
-```
 
-Disables optimization for a rule.
+#### NOT_OPTIMIZED hint
+By gathering anonymous statistics on used filtering rules, we can detect and remove the rules that are no longer used. More information about rules optimization you may find in the following article: <https://adguard.com/en/filter-rules-statistics.html>.
```

Comment:
For large filters, AdGuard compiles two versions: full and optimized. Optimized version is much smaller and does not contain rules which are not used at all or used rarely. More information about rules optimization you may find in [this article](https://adguard.com/en/filter-rules-statistics.html).